cmd/roles/roles.go

```diff
@@ -324,6 +324,7 @@ func (mr *MilvusRoles) Run(alias string) {
 		paramtable.Init()
 	}
 
+	mr.setupLogger()
```

Comment:
Moving `setupLogger` here will cause `nodeID` @Line#226 always be 0. Is that ok?

Reply:
It looks like this way is impassable.